packages/oca/src/interfaces/data/branding/BrandingOverlayData.interface.ts

```diff
@@ -4,7 +4,7 @@ export default interface IBrandingOverlayData extends IBaseOverlayData {
   logo: string
   background_image: string
   background_image_slice: string
-  primary_background_color: string
+  primary_background_color?: string
```

Comment:
Any reason this is now optional or is it a carry over from something else?

Reply:
Could we get around this by using a conditional OR instead on line 22? (i.e. `||`)